docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -144,6 +145,17 @@ In order to get the MSI, your proxy needs to allow HTTPS connections to the foll
 * `https://aka.ms/`
 * `https://azcliprod.blob.core.windows.net/`
 
+### Migrate to 64-bit Azure CLI
+Starting from 2.51.0, Azure CLI also provides 64-bit version which is recommended for better performance.
+
+If you are using 32-bit version, you can migrate to 64-bit version by following these steps:
+1. Check your current CLI version and installed extensions by running `az --version`.
+2. If the extension folder `%userprofile%\.azure\cliextensions` exists, backup it by renaming. This folder will be created again after migration.
+3. Download and install latest 64-bit version following the [Install or update](#install-or-update) guide. Please note that 32-bit version will be uninstalled automatically.
+4. Install extensions by running `az extension add --name <extension> --version <version>`.
```

Comment:
**Question 1:** Our extensions automatically install as 32 or 64 depending on the Core installation?
**Question 2:** When moving up (32-->64) or moving back (64-->32) are all existing extensions automatically reinstalled?
**Question 3:** What happens when a customer is running 64-bit, and for whatever reason, overlays their 64-bit extension folder with their 32-bit extension backup? Is there an expected error message when there is a mismatch?

Reply:
Question 4: Is installation of 64bit on top of 32 bit supported? If not, this needs to be clearly mentioned.

Reply:
Question 1: Yes
Question 2: No, users have to manually reinstalled the extensions.
Question 3: No, the error will be raised during command execution. We don't know if the dependencies are 32-bit or 64-bit.
Question 4: Yes, 32-bit is uninstalled automatically when installing 64-bit.

Reply:
> Question 4: Is installation of 64bit on top of 32 bit supported? If not, this needs to be clearly mentioned.

@dcaro, I put a sentence on line 42, and @bebound repeated this information on line 157 and 160.  Let me rework line 27.